Library/Formula/swig.rb

```diff
@@ -12,6 +12,8 @@ class Swig < Formula
 
   option :universal
 
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!